docs/pages/admin-guide.mdx

```diff
@@ -512,61 +468,62 @@ it can also generate a new token for you). In the following example a new token
 is created with a TTL of 5 minutes:
 
 ```bash
-$ tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
+tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
 The invite token: secret-value
 ```
+{/* Convert to new UI component https://github.com/gravitational/next/issues/275 */}
 
 If `--token` is not provided, [`tctl`](cli-docs.mdx#tctl) will generate one:
 
 ```bash
-# generate a short-lived invitation token for a new node:
-$ tctl nodes add --ttl=5m --roles=node,proxy
+# Generate a short-lived invitation token for a new node:
+tctl nodes add --ttl=5m --roles=node,proxy
```

Comment:
```suggestion
tctl nodes add --ttl=5m --roles=node,app
```
How about we make these more focused on adding servers + apps on a host. Vs the lesser used adding a proxy host?